dtos/requests/intervalaction_test.go

```diff
@@ -90,7 +90,7 @@ func TestAddIntervalActionRequest_Validate(t *testing.T) {
 		{"valid AddIntervalActionRequest", valid, false},
 		{"valid AddIntervalActionRequest, no Request Id", noReqId, false},
 		{"valid AddIntervalActionRequest, interval name containing unreserved chars", intervalNameWithUnreservedChars, false},
-		{"invalid AddIntervalActionRequest, interval name containing reserved chars", intervalNameWithReservedChars, true},
+		{"Valid AddIntervalActionRequest, interval name containing reserved chars", intervalNameWithReservedChars, false},
```

Comment:
lowercase to meet the original test message

Reply:
Updated.